docs/_static/js/copycode.js

```diff
@@ -62,6 +62,13 @@ $(document).ready(function(){
         e.clearSelection();
 
         for(var i = 0; i < lines.length; ++i) {
+            for(var lang in LANG_GP) {
+                if(lines[i].indexOf(LANG_GP[lang]) != -1) {
+                    var re = new RegExp(LANG_GP[lang], g);
+                    lines[i] = lines[i].replace(re, "");
+                    break;
+                }
+            }
             lines[i] = lines[i].replace(/^\s+|\s+$/g, "");
```

Comment:
I know you didn't change this line, but it seems to be throwing errors in the preview.
```
Uncaught ReferenceError: g is not defined
    at copycode.js:67
    at t.emit (clipboard.min.js:7)
    at e (clipboard.min.js:7)
    at e (clipboard.min.js:7)
    at e (clipboard.min.js:7)
    at e (clipboard.min.js:7)
    at new e (clipboard.min.js:7)
    at t.e (clipboard.min.js:7)
    at HTMLBodyElement.<anonymous> (clipboard.min.js:7)
    at HTMLBodyElement.<anonymous> (clipboard.min.js:7)
```
There's also some 403 errors happening which really are 404's.  Seems like there might be a bigger issue going on here...

Reply:
Actually, it is a line in the updated code block... line 67...
```
var re = new RegExp(LANG_GP[lang], g);
```

Reply:
Fixed.